stylesheets/_component.panels.scss

```diff
@@ -108,8 +115,8 @@
 }
 
 .panel--padded {
-    padding-top: 24px;
-    padding-bottom: 24px;
+    padding-top: ($padding-large-vertical * 2);
```

Comment:
Properties within rules should be alpha ordered.

Reply:
Should probably add an automated test for this 😄

Reply:
Fixed